src/queries/a11yState.ts

```diff
@@ -10,6 +10,23 @@ import type {
   QueryByQuery,
 } from './makeQueries';
 
+export function matchAccessibilityState(
+  node: ReactTestInstance,
+  matcher: AccessibilityState
+) {
+  const stateProp = node.props.accessibilityState;
+
+  // busy, disabled & selected states default to false,
+  // while checked & expended states treat false and default as sepatate values
+  return (
```

Comment:
I think that the default values are a bit implicit here (hence the need for a comment), wdyt about having an object for those default values ? 

```tsx

const DEFAULT_ACCESSIBILITY_VALUES: Record<
  keyof AccessibilityState,
  boolean | undefined
> = {
  busy: false,
  disabled: false,
  selected: false,
  checked: undefined,
  expanded: undefined,
};

export function matchAccessibilityState(
  node: ReactTestInstance,
  matcher: AccessibilityState
) {
  const stateProp = node.props.accessibilityState;
  // busy, disabled & selected states default to false,
  // while checked & expended states treat false and default as sepatate values
  return (
    matchState(
      stateProp?.busy ?? DEFAULT_ACCESSIBILITY_VALUES.busy,
      matcher.busy
    ) &&
    matchState(
      stateProp?.disabled ?? DEFAULT_ACCESSIBILITY_VALUES.disabled,
      matcher.disabled
    ) &&
    matchState(
      stateProp?.selected ?? DEFAULT_ACCESSIBILITY_VALUES.selected,
      matcher.selected
    ) &&
    matchState(
      stateProp?.checked ?? DEFAULT_ACCESSIBILITY_VALUES.checked,
      matcher.checked
    ) &&
    matchState(
      stateProp?.expanded ?? DEFAULT_ACCESSIBILITY_VALUES.expanded,
      matcher.expanded
    )
  );
}

```

Something like that for instance ? I also think that there should be a way to refactor the calls to matchState but I don't know how to map on Accessibility states properly with correct typing